src/process_utility.c

```diff
@@ -555,7 +637,11 @@ process_grant_and_revoke(ProcessUtilityArgs *args)
 
 	switch (stmt->objtype)
 	{
+#if PG96 || PG10
```

Comment:
Unfortunately, that doesn't work, because they are consolidating on an existing OBJECT_TABLESPACE thing that did exist previously, but wasn't being used here, so we'd be overloading OBJECT_TABLESPACE, unless I misunderstand what you're saying.

Reply:
At least that's what I remember when I reviewed the commit that caused this change. I need to find it again and will add it here in a comment.